services/namespaces/service.go

```diff
@@ -99,13 +105,24 @@ func (s *Service) Create(ctx context.Context, req *api.CreateNamespaceRequest) (
 
 		resp.Namespace = req.Namespace
 		return nil
-	})
+	}); err != nil {
+		return &resp, err
+	}
+
+	if err := s.emit(ctx, "/namespaces/create", event.NamespaceCreate{
```

Comment:
Namespaces can also be created implicitly, when in use. This makes it easy to just start using the API, but we never call into the a namespace service or store to make this happen.